### ISSUE TITLE
javaagent requires absolute path to GAE SDK

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tsaiyilien/google-plugin-for-eclipse#226]
```
I would like to have a run configuration that can be shared across machines 
without requiring an absolute path to the GAE sdk.

Create a run configuration for a 'Web Application'

Attempt to set the -javaagent parameter to something other than the default 
setting.

Before: http://take.ms/cqza1C
After: http://take.ms/FJ1Y5

```

Original issue reported on code.google.com by `latch...@gmail.com` on 23 Jul 2013 at 5:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `nhcohen@google.com` on 23 Jul 2013 at 1:16

**Comment by GoogleCodeExporter** [comment]
```
We are comparing the javaagent VM argument in the launch-configuration dialog 
with the expected value without expanding ${...} variables.
```

Original comment by `nhcohen@google.com` on 23 Jul 2013 at 10:26
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for picking this one up so quickly. Yea, being able to expand the ${} 
and not requiring the GAE SDK to be in a particular location would be super 
helpful. Just validate that I'm pointing at a appengine-agent.jar file 
somewhere on the filesystem would be optimal. If you notice in the screenshot, 
it seems like the other parameter (-Xbootclasspath...) works ok with the ${} 
expansion.
```

Original comment by `latch...@gmail.com` on 24 Jul 2013 at 2:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdayal@google.com` on 12 Aug 2013 at 10:26
- Added labels: **Release-3.4.0**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdayal@google.com` on 20 Dec 2013 at 9:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Running Eclipse 4.4 and plugin version 3.8.0 I see this same issue.  It looks 
like this was fixed in 3.4.0.  While the variable does indeed work (yay), the 
warning in the argument configuration still exists and re-opening the launch 
profile expands the variable, effectively editing the launch file again.
```

Original comment by `knitt...@razzle.mobi` on 28 Feb 2015 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
I long ago gave up trying to run in Eclipse. I *highly* suggest you switch to 
IDEA. The AppEngine integration there is far better and full of fewer headaches 
and IDEA is generally a better IDE anyway.
```

Original comment by `latch...@gmail.com` on 28 Feb 2015 at 9:15
